css-color-4/Overview.bs

```diff
@@ -4147,7 +4152,8 @@ Predefined Color Spaces</h2>
 
 		</dl>
 
-<h3 id="predefined-a98-rgb">Predefined a98-rgb space</h3>
+<h3 id="predefined-a98-rgb">
+The Predefined ??? Color Space: the ''a98-rgb'' keyword</h3>
```

Comment:
Yeah, trademark law is fun. I guess just put A98 RGB

Reply:
What about calling it opRGB? https://en.wikipedia.org/wiki/Adobe_RGB_color_space

Reply:
(replied offline)